### PR TITLE
PT-420 Update Build process

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
- Remove yarn.lock (unused since we use npm)
- Upgrade nodejs to v10.15.3 ?
- Missing CHANGELOG.md
- Use npm in Dockerfile
- Missing config/env.sh ?

File: Dockerfile [hunk 1]
```diff
@@ -8,19 +8,19 @@ COPY package.json /app
 # ---- Dependencies ----
 FROM base AS dependencies
 # install only production node packages
-RUN yarn install --production
+RUN npm install --production
```
Comment (yvon-dblg) [inline Dockerfile:11]:
```suggestion
RUN npm install --only=production
```

File: build.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 rm -rf node_modules
 
 TIME="$(date +%s)"
-yarn --production
+npm install --production
```
Comment (yvon-dblg) [inline build.sh:6]:
```suggestion
npm install --only=production
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Export
+Service to export YourLoops data in selected formats, either csv, json or xls.
+
+## [1.3.2] - 2019-09-10
```
Comment (yvon-dblg) [inline CHANGELOG.md:4]: Should we use our version number ?
1.3.2 is the tidepool version.

**Review by yvon-dblg — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (franck-fourel) [reply Dockerfile:11]: According to the documentation, the `--production` is ok

File: build.sh (hunk 2 above)
Comment (franck-fourel) [reply build.sh:6]: According to the documentation, the `--production` is ok

File: CHANGELOG.md (hunk 3 above)
Comment (franck-fourel) [reply CHANGELOG.md:4]: Let's use the 1.3.2 for now.

**Review by yvon-dblg — APPROVED** [review]

**Pull request merged**